README.md

```diff
@@ -101,13 +101,9 @@ More info [here](https://packages.gentoo.org/packages/sys-apps/moar).
 
 ## Using [Arch Linux](https://archlinux.org/)
 
-:warning: Installs legacy `moar` binary.
-
-```sh
-yay -S moar
-```
+Install `moor` with your [AUR helper](https://wiki.archlinux.org/title/AUR_helpers) of choice or follow the instructions [here](https://wiki.archlinux.org/title/Arch_User_Repository) to install the official way.
 
-More info [here](https://aur.archlinux.org/packages/moar-git).
+More info [here](https://aur.archlinux.org/packages/moor).
```

Comment:
Following that link, there's a URL in there that should be updated to point to https://github.com/walles/moor rather than the old URL.